Function Attributes/demo/template-inference/tempinf.d

```diff
@@ -0,0 +1,16 @@
+void takePointer(int*) @safe {}
```

Comment:
Is this exercise tied to any section? It doesn't seem so...

Reply:
Sorry, I forgot. Will do.